lib/wprof/reporters/file_report.rb

```diff
@@ -0,0 +1,63 @@
+module Wprof
+  module Reporters
+    module FileReport
+      def file_report
```

Comment:
Beware of this method name, as above it's kind of redundant to have a method with the same name as the object a better call would be `generate` or something similar

Reply:
Done.